### PR TITLE
Fix stm32 paren warns

### PR DESCRIPTION
**Pull request by galak**
Fix warnings reported by clang in the STM32 HAL code

### CODE REVIEW

**Comment by galak** [comment]
@erwango let me know what other changes you need for this w/regards to README files, etc.

File: ext/hal/st/stm32cube/stm32l4xx/drivers/src/stm32l4xx_hal_tim_ex.c [hunk 1]
```diff
@@ -406,11 +406,11 @@ HAL_StatusTypeDef HAL_TIMEx_HallSensor_Start_DMA(TIM_HandleTypeDef *htim, uint32
   /* Check the parameters */
   assert_param(IS_TIM_HALL_SENSOR_INTERFACE_INSTANCE(htim->Instance));
 
-  if ((htim->State == HAL_TIM_STATE_BUSY))
+  if ((htim->State == HAL_TIM_STATE_BUSY)
```
Comment (ulfalizer) [inline ext/hal/st/stm32cube/stm32l4xx/drivers/src/stm32l4xx_hal_tim_ex.c]: Forgot to remove one of them? Same in rest of file.

**Review by erwango — CHANGES_REQUESTED** [review]
Please update packages README files. Will provide you the bugtracker refs.

**Comment by erwango** [comment]
And here you go:
F0: 63613
F1: 63614
F2: 63615
F3: 63616
F4: 63617
F7: 63618
L0: 63619
L4: 63620

**Review by ulfalizer — APPROVED** [review]

**Review by tyrantlizard77 — APPROVED** [review]

**Review by erwango — APPROVED** [review]

**Pull request merged**